src/core/image_resizer.js

```diff
@@ -160,10 +167,33 @@ class ImageResizer {
 
   async _createImage() {
     const data = this._encodeBMP();
-    const blob = new Blob([data.buffer], {
-      type: "image/bmp",
-    });
-    const bitmapPromise = createImageBitmap(blob);
+
+    let decoder, imagePromise;
+
+    // TODO: remove the isChrome, once Chrome isn't crashing anymore with
+    // isssue6741.pdf.
+    if (
+      (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) ||
```

Comment:
Unfortunately I don't believe that enabling this unconditionally in Firefox is safe, since this can be disabled with a preference *and* that preference is only enabled by default in Nightly: https://searchfox.org/mozilla-central/rev/ec342a3d481d9ac3324d1041e05eefa6b61392d2/modules/libpref/init/StaticPrefList.yaml#3241-3245